tests/e2e/etcd_config_test.go

```diff
@@ -16,18 +16,19 @@ package e2e
 
 import (
 	"fmt"
+	"github.com/stretchr/testify/assert"
 	"io/ioutil"
 	"os"
 	"strings"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
```

Comment:
Hey @serathius - Nitpick - when I run `go run golang.org/x/tools/cmd/goimports -w tests/e2e/etcd_config_test.go` on this it is suggesting the following:

```diff
 import (
        "fmt"
-       "github.com/stretchr/testify/assert"
        "io/ioutil"
        "os"
        "strings"
        "testing"
 
+       "github.com/stretchr/testify/assert"
+
        "go.etcd.io/etcd/pkg/expect"
 )
```